src/Orchard.Environment.Extensions/Loaders/DynamicExtensionLoader.cs

```diff
@@ -66,7 +66,7 @@ public class DynamicExtensionLoader : IExtensionLoader {
                 }
                 catch (System.Exception ex) {
                     _logger.LogError(string.Format("Error trying to load extension {0}", descriptor.Id), ex);
-                    throw;
+                    return null;
```

Comment:
System.Exception can not be handled this way; impossible to know what's going on.
Let it break or handle it, right? https://msdn.microsoft.com/en-us/library/x4cw969y(v=vs.110).aspx